packages/types/src/lib/interfaces.ts

```diff
@@ -255,6 +256,14 @@ export interface WithSessionSigs extends BaseJsonExecutionRequest {
 
 export type JsonExecutionRequest = WithAuthSig | WithSessionSigs;
 
+export interface JsExecutionRequestBody {
```

Comment:
Similar changes should be made to the pkpSign interface: https://github.com/LIT-Protocol/lit-assets/blob/aff82c4381139476aeaa6faa814069b184d38d39/rust/lit-node/src/endpoints/versions/v1.rs#L101